plugin/src/main/java/org/opensearch/ml/action/deploy/TransportDeployModelAction.java

```diff
@@ -405,4 +438,10 @@ void updateModelDeployStatusAndTriggerOnNodesAction(
             );
     }
 
+    // this method is only to stub static method.
```

Comment:
Why need to do this?

Reply:
Yeah, so that I can stub the value in the method like this: `doReturn(true).when(transportDeployModelAction).isSuperAdminUserWrapper(clusterService, client);`